app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala

```diff
@@ -64,11 +66,29 @@ class ResponseBuilder[F[_] : ME : Logger] {
     asGetEditSetRecordRequest(jmsMessage).flatMap(getEditSetRecordRequest =>
       getEditSetRecordByOCI(getEditSetRecordRequest.recordOci)
         .map(editSetRecord => me.pure(Json.toJson(editSetRecord).toString))
-        .getOrElse(
-          onUnhandledCase(
-            s"Unable to find record for Edit Set with OCI [${getEditSetRecordRequest.editSetOci}] and Record OCI [${getEditSetRecordRequest.recordOci}]"
-          )
+        getOrElse onUnknownEditSetRecord(getEditSetRecordRequest.editSetOci, getEditSetRecordRequest.recordOci)
```

Comment:
The method call syntax doesn't match the rest of the project here, elsewhere we use the `.` syntax.

Reply:
Sorry, how do you mean?

Reply:
Oh - I'm missing the dot before `getOrElse` - odd that it works. Will change.

Reply:
Changed. I was surprised that it compiled.